textproc/cloc/Portfile

```diff
@@ -4,7 +4,7 @@ PortSystem          1.0
 PortGroup           github 1.0
 PortGroup           perl5 1.0
 
-github.setup        AlDanial cloc 1.92 v
+github.setup        AlDanial cloc 1.94 v
 revision            1
```

Comment:
When bumping the version, `revision` should reset back to zero.

Reply:
Thanks for pointing this out! I just set the `revision` field to zero.